.github/workflows/build_canary.yml

```diff
@@ -0,0 +1,89 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish canary image to GitHub Container Registry
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build_operator:
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
can you fix this to `ubuntu-20.04`? nothing will change from under us if/when the latest changes in the future. same with the others below